### PR TITLE
Embed Videos for iOS tutorials

### PR DESCRIPTION
**Pull request by abdulajet**
## Description

Adding a link and embed for the iOS Core SDK tutorials. Size is changed to fit mobile layout

https://nexmo-developer-pr-3931.herokuapp.com/client-sdk/tutorials/phone-to-app/introduction/swift
https://nexmo-developer-pr-3931.herokuapp.com/client-sdk/tutorials/app-to-phone/introduction/swift
https://nexmo-developer-pr-3931.herokuapp.com/client-sdk/tutorials/app-to-app/introduction/swift

### CODE REVIEW

**Review by pardel — CHANGES_REQUESTED** [review]
Can we make the video embeds a bit bigger? Also, for the link to 'YouTube', could we open in new window so people will end up with both video & text versions open at the same time?

**Comment by abdulajet** [comment]
> Can we make the video embeds a bit bigger? 

I couldn't make the iframe responsive so that's the biggest size I could fit on mobile with the correct padding. Here is a page with the original size https://developer.vonage.com/voice/sip/configure/genesyscloud

Happy for a more seasoned CSS wrangler to take a look

File: _documentation/en/voice/voice-api/guides/ncco.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ description: >-
 A Call Control Object (NCCO) is a JSON array of actions that is used to control the flow of a Voice API Call.
 
 Below you can find the One Dev Minute video explaining what NCCOs are and how to use them:
-<center><iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/26rm0MP2up0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></center>
+<center><iframe width="448" height="252" src="https://www.youtube-nocookie.com/embed/26rm0MP2up0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></center>
```
Comment (conshus) [inline _documentation/en/voice/voice-api/guides/ncco.md:15]:
```suggestion
<center style="aspect-ratio: 16 / 9;"><iframe width="100%" height="100%" src="https://www.youtube-nocookie.com/embed/26rm0MP2up0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></center>
```

File: config/tutorials/en/app-to-app/swift.yml [hunk 2]
```diff
@@ -15,6 +15,10 @@ introduction:
     * An iOS application that will make the voice call using the Client SDK.
     * A publicly accessible URL that serves instructions on how to handle the call. To accomplish this, you will create a webhook server using <a href="https://nodejs.org/en/" target="_blank">Node.js</a>, <a href="https://expressjs.com/" target="_blank">Express</a> and <a href="https://github.com/localtunnel/localtunnel" target="_blank">localtunnel</a>.
 
+    ## Video tutorial
+    If you prefer to follow along by video, you can find this tutorial on <a href="https://www.youtube.com/watch?v=3EwAAKYnkm4" target="_blank">YouTube</a>:
+    <center><iframe width="448" height="252" src="https://www.youtube-nocookie.com/embed/3EwAAKYnkm4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></center>
```
Comment (conshus) [inline config/tutorials/en/app-to-app/swift.yml:20]:
```suggestion
    <center style="aspect-ratio: 16 / 9;"><iframe width="100%" height="100%" src="https://www.youtube-nocookie.com/embed/3EwAAKYnkm4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></center>
```

File: config/tutorials/en/app-to-phone/swift.yml [hunk 3]
```diff
@@ -15,6 +15,10 @@ introduction:
     * An iOS application that will make the voice call using the Client SDK.
     * A publicly accessible URL that serves instructions on how to handle the call. To accomplish this, you will create a webhook server using <a href="https://nodejs.org/en/" target="_blank">Node.js</a>, <a href="https://expressjs.com/" target="_blank">Express</a> and <a href="https://github.com/localtunnel/localtunnel" target="_blank">localtunnel</a>.
 
+    ## Video tutorial
+    If you prefer to follow along by video, you can find this tutorial on <a href="https://www.youtube.com/watch?v=s2MjXXIueoo" target="_blank">YouTube</a>:
+    <center><iframe width="448" height="252" src="https://www.youtube-nocookie.com/embed/s2MjXXIueoo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></center>
```
Comment (conshus) [inline config/tutorials/en/app-to-phone/swift.yml:20]:
```suggestion
    <center style="aspect-ratio: 16 / 9;"><iframe width="100%" height="100%" src="https://www.youtube-nocookie.com/embed/s2MjXXIueoo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></center>
```

**Review by conshus — COMMENTED** [review]
I added some code to make the videos responsive to 16:9 aspect ratio. It's supported in modern browsers except IE: https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio

Try it out and let me know if it works the way you wanted.

If we want, we can go back to the old hack of adding padding.

**Pull request merged**